src/ppspp_handshake.erl

```diff
@@ -89,7 +94,8 @@ handle({handshake, Body}, Endpoint) ->
         % prove useful to simply create a new channel anyway, which would
         % allow peers that dropped off for some reason to restart cleanly.
         false -> {ok, _Pid} =
-                 channel_worker:where_is(Inbound_channel)
+                 channel_worker:where_is({channel,Inbound_channel})
     end,
     % send handshake message to channel_worker for reply
-    channel_worker:handle(Pid, {handshake, Source_channel}).
+    ok = channel_worker:handle(Pid, #{handshake => #{channel => Source_channel}}),
```

Comment:
Line 100 is too long:     ok = channel_worker:handle(Pid, #{handshake => #{channel => Source_channel}}),.